gfx/NeoDays/pngs_tiles_10x10/mod/magiclysm/monsters/goblins/goblins.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "id": "mon_goblin_warrior",
+    "fg": "mon_goblin_warrior"
```

Comment:
Don't need the json if "id" and "fg" are the same, basically calling the png file with the id of the thing they represent is enough

Reply:
got it.  I matched all png filenames with the monsters they represented, so I will remove the json documents.